### PR TITLE
LPS-85269 Enable service builder export packages checker

### PR DESCRIPTION
**Pull request by Camork**

### CODE REVIEW

File: modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/bnd.bnd [hunk 1]
```diff
@@ -6,4 +6,5 @@ Export-Package:\
 	${package}.model,\
 	${package}.service,\
 	${package}.service.persistence
--includeresource: META-INF/service.xml=../${artifactId}-service/service.xml
\ No newline at end of file
+-includeresource: META-INF/service.xml=../${artifactId}-service/service.xml
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/bnd.bnd]: hey @Camork  can you sort these headers please?  thanks!

Comment (Camork) [reply modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/bnd.bnd]: OK.

File: modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/bnd.bnd [hunk 2]
```diff
@@ -6,4 +6,5 @@ Export-Package:\
 	${package}.model,\
 	${package}.service,\
 	${package}.service.persistence
+-check: EXPORTS
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/bnd.bnd:9]: Hey @Camork  Can you add a unit test that will create a project, change the package in the service.xml and run the 'buildService' task and then try to build the jar and assert that the build fails with the export error?

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/petershin/liferay-portal/pull/220. [See changes here.](https://github.com/gamerson/liferay-portal/compare/4af164149f3d~4...4af164149f3d) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**